### PR TITLE
Pro Track Challenge Page Designs

### PR DESCRIPTION
**Pull request by miguelaenlle**
- Implemented designs for the Pro Track Challenge description page:
image

- Implemented Challenge Passed page design:
image

- Implemented Challenge Not Passed page design:
image

Todo:
- Make the navbar dark only on the Challenge Description page ✅ Done
- Mobile optimizations for these pages ✅ Done
- Implement designs for "Sign Up As" / "Are you worthy of the title HackOlympian" pages (if needed and if designs are available)

### CODE REVIEW

File: components/Challenge/ProChallengeContent.tsx [hunk 1]
```diff
@@ -41,144 +48,198 @@ const ProChallengeContent: React.FC<ProChallengeContentProps> = ({
...
-                    text="Finished Challenge"
-                    onClick={handleUserFinishedChallenge}
+                <div className={styles.links}>
+                    <p onClick={handleSuccess} className={styles.link}>
+                        <a>Test success case</a>
+                    </p>
+                    <p onClick={handleFailure} className={styles.link}>
+                        <a>Test failure case</a>
+                    </p>
+                </div>
```
Comment (J164) [inline components/Challenge/ProChallengeContent.tsx]: This probably shouldn't go into production

**Review by J164 — CHANGES_REQUESTED** [review]
I get a 400 error when the website hits artemis.hackillinois.org/status because the endpoint requires that the user made a GET request to artemis.hackillinois.org/challenge first. This could probably be fixed by making that GET request to /challenge when the user clicks "Begin Challenge"

File: components/Challenge/ProChallengeContent.tsx (hunk 1 above)
Comment (miguelaenlle) [reply components/Challenge/ProChallengeContent.tsx]: I left this for testing purposes but feel free to remove it / let me know to remove it when ready

**Review by J164 — APPROVED** [review]
Great work on this!

**Pull request merged**